.gitignore

```diff
@@ -79,7 +79,6 @@ $RECYCLE.BIN/
 *#
 .idea
 nbproject
-.vscode/
```

Comment:
Because vscode bases it's eslint on the root dir when it has to base it on each package folder. I wouldn't put too many IDE things in there but just config that makes it work :)

Reply:
I'm just a bit afraid because it means that everyone using vs code will have their local workspace config in the untracked files (& won't be ignored)

Reply:
I can change the ignore rule to only keep that file and ignore any other file in the folder but there is nothign wrong having a sharde config for common IDEs

Reply:
I'm not really against having a shared config, but in some of their local configs, developers might have hardcoded tokens or secrets (in the run config env or elsewhere) and it would make it easy to forget about that and push all the changes.
I believe having very precise rules regarding this would be safer. What do you think?

Reply:
If the files where shared you wouldn't have your own so there wouldn't be the problem you mention. But that's not the goal here anyway I'll just ignore everything except the settings file which can never contain secrets or run configs.